src/components/Header/Header.js

```diff
@@ -0,0 +1,28 @@
+import styled from 'styled-components';
+
+export default function Header() {
+  return (
+    <HeaderContainer>
+      <span>JOYFEEL WOK</span>
+    </HeaderContainer>
+  );
+}
+
+const HeaderContainer = styled.div`
```

Comment:
I think you could be more semantic here. O do you have a reason why you are not working with a `header` element? And you could also use a `h1` element instead of a `span` element I suppose. :)

Reply:
Ah no, I think this was misunderstanding. I mean you can use the `header` element instead of a `div` here to be more semantic.